integration/tests/cook/util.py

```diff
@@ -199,6 +199,21 @@ def query_jobs(cook_url, assert_response=False, **kwargs):
     return response
 
 
+def query_jobs(cook_url, assert_response=False, **kwargs):
```

Comment:
Should we also introduce a `query_job` function that hits the `/jobs/<uuid>` endpoint?

Reply:
`load_job` hits `/jobs/<uuid>`